### DISCUSSION TITLE
Want to integrate woocommerce with etsy. Sync orders, products etc

### DISCUSSION THREAD

**Discussion by anshu0220593** [etsy/open-api#1151] [Q&A] [answered]
Hi Community, 

I am using Ecart API. Does anyone work with it?

The below tasks are completed by me.

**1) Collect product price, quantity, etc meta info 
2) Get access token dynamically from the product page as a chosen vendor 
3) Insert Product on ETSY store for vendor  
4) Update Product on the ETSY store for vendor**

But I got stuck now. Actually, I want to implement the below tasks now using Ecart API.

**5) Update Product on the ETSY store for vendor 
6) Sync orders from the ETSY store to woocommerce 
7) Push Shipping status on the ETSY store from woocommerce** 

To achieve 5) points I want to trigger webhook first through Etsy whenever a new order is placed. 

How can I do it? Can you provide documentation to do so, instructions, or some visual media sets to do so? 

Kindly help me here.

After 5 I go towards 6 and 7 points also.

Kindly help me here. Waiting for your response.

Thanks,
Anshul Singh

**Comment by Gareth064** [comment]
The short answer is, there are no webhooks available for Etsy. You have to poll Etsy to check for new orders.

**Reply by anshu0220593** [reply]
I am not convinced by your comment. Etsy provides API to get invoice info.
See link: https://developers.etsy.com/documentation/reference/#operation/getShopReceipt

**Reply by Gareth064** [reply, marked as answer]
Correct. You have to call that API endpoint to get orders.

But Etsy doesn't give you a webhook endpoint to subscribe to, which will tell you when a new order comes in. Which is what I believe you asked about.
